src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -119,16 +114,22 @@ public void run() {
                         .setTaskId(task.getTaskId());
                 try {
                     NMTaskConfig taskConfig = GSON.fromJson(task.getData().toStringUtf8(), NMTaskConfig.class);
-                    LOGGER.info("TaskConfig: ", taskConfig);
-                    ProcessBuilder processBuilder = buildProcessBuilder(task, taskConfig);
-                    MyriadExecutor.this.process = processBuilder.start();
+                    if (!permisionsSet && taskConfig.getRemoteDistribution() && !setPermissions(taskConfig)) {
+                        LOGGER.info("Trying to use remote URI but can't set permissions, " +
```

Comment:
Maybe log the user:password (or entire command string), in case this errors due to an invalid user/group, or other error besides lack of permissions.